src/components/__stories__/ColumnPinningDemo.classname.ts

```diff
@@ -1,3 +1,3 @@
-import {block} from '../../utils';
+import {cn} from '../../utils';
 
-export const cnColumnPinningDemo = block('column-pinning-demo');
+export const cnColumnPinningDemo = cn('column-pinning-demo');
```

Comment:
Why was this change necessary?

Reply:
There are no reasons to add `gt-` prefix for demo components.
That's why we use `cn` in stories.